Header header
string[] joint_name
float64[] joint_position
---
geometry_msgs/PoseStamped end_effector_pose
bool success
